ddapm_test_agent/agent.py

```diff
@@ -469,6 +469,15 @@ async def handle_snapshot(self, request: Request) -> web.Response:
         span_ignores = list(default_span_ignores | overrides)
         log.info("using ignores %r", span_ignores)
 
+        # Get the span attributes that are to be removed for this snapshot.
+        default_span_removes: Set[str] = request.app["snapshot_removed_attrs"]
+        overrides = set(_parse_csv(request.url.query.get("removes", "")))
+        span_removes = list(default_span_removes | overrides)
+        log.info("using removes %r", span_removes)
+
+        if "start" in span_removes and "span_id" in span_removes:
```

Comment:
```suggestion
        if "span_id" in span_removes:
```

I think we can simplify this condition to just `span_id`. The span id is normalized (based off the `start` attribute) in the snapshot. It is used to assert the position and order of each span in the trace which makes it pretty crucial for the snapshot testing. Its value should be stable so I don't see a use-case where someone would want to ignore it.